src/core.js

```diff
@@ -755,11 +755,30 @@ define([
             }
 
             if (window.analytics) {
-                window.analytics.usage(
-                    "Question Reference",
-                    "Drag and Drop",
-                    _this.data.core.currentlyEditedProperty
-                );
+                var targetType;
+                switch (target[0].id) {
+                    case 'property-relevantAttr':
+                        targetType = "Display";
+                        break;
+                    case 'property-constraintAttr':
+                        targetType = "Validation";
+                        break;
+                    case 'property-calculateAttr':
+                        targetType = "Calculation";
+                        break;
+                    default:
+                        targetType = "Expression Editor";
+                        break;
+                }
+                if (path.contains('casedb')) {
```

Comment:
the problem with `instance('casedb')` is that both single and double quotes are allowed (not too hard to work around with a regexp) and that shorthand hashtag paths are coming (see the PR I linked above), which do not use that syntax or even contain the substring "casedb". Once hashtag paths become a thing I think we'll want utility functions for classifying paths rather than ad-hock pattern matching scattered throughout the codebase.